pyproject.toml

```diff
@@ -0,0 +1,156 @@
+# https://peps.python.org/pep-0517/
+[build-system]
+requires = ["setuptools>=45", "wheel"]
+build-backend = "setuptools.build_meta"
+
+# https://peps.python.org/pep-0621/
+[project]
+name = "napari-aicsimageio"
+# Do not edit this string manually, always use bumpversion
+# Details in CONTRIBUTING.rst
+version = "0.7.0"
```

Comment:
Assuming that if we swap to setuptools_scm this changes as well.

Reply:
correct, one would then add "version"  to the `dynamic` section below: https://peps.python.org/pep-0621/#dynamic

Reply:
I'm kinda inclined to put that in a different PR, so you can see those changes isolated.  sound good?

Reply:
Works for me!